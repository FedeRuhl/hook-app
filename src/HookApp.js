import React from 'react';

const HookApp = () => {
    return (
        <>
            <h1>
                Test
            </h1>   
        </>
    )
};

export default HookApp;