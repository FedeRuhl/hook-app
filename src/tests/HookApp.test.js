import { shallow } from 'enzyme';
import React from 'react';
import HookApp from '../HookApp';

describe('Hook App tests', () => {

    test('should show the componente correctly', () => {
        const wrapper = shallow(<HookApp />);
       expect(wrapper).toMatchSnapshot(); 
    });
    
});