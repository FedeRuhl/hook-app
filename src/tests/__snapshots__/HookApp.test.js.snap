// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hook App tests should show the componente correctly 1`] = `
<Fragment>
  <h1>
    Test
  </h1>
</Fragment>
`;
